cooperator_website/controllers/main.py

```diff
@@ -201,16 +209,11 @@ def fill_values(self, values, is_company, logged, load_from_user=False):
                     break
             if not values.get("share_product_id", False) and products:
                 values["share_product_id"] = products[0].id
-        if not values.get("country_id"):
-            if company.default_country_id:
-                values["country_id"] = company.default_country_id.id
-            else:
-                values["country_id"] = "20"
         if not values.get("activities_country_id"):
             if company.default_country_id:
                 values["activities_country_id"] = company.default_country_id.id
             else:
-                values["activities_country_id"] = "20"
+                values["activities_country_id"] = 20
```

Comment:
this part is obsolete (i didn’t know about this field and it is used nowhere anymore) and can be removed.